projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/case-file-view-field.component.spec.ts

```diff
@@ -233,4 +242,25 @@ describe('CaseFileViewFieldComponent', () => {
     const errorMessagesFromElements = listElements.map(item => item.nativeElement.textContent);
     expect(component.errorMessages).toEqual(errorMessagesFromElements);
   });
+
+  it('should enable icp with test config', () => {
+    fixture.detectChanges();
+    component.isIcpEnabled();
+    expect(component.icpEnabled).toBeTruthy();
```

Comment:
This isn't quite right. component.icpEnabled is initialised by ngOninit, so there is no point in calling component.isIcpEnabled because it has no side effects.